deploy/template.yaml

```diff
@@ -34,14 +34,14 @@ Resources:
         Variables:
           ENVIRONMENT: !Sub "${Environment}"
           USER_ISSUED_CREDENTIALS_TABLE_NAME: !Select [1, !Split ['/', !GetAtt UserIssuedCredentialsTable.Arn]]
-          SHARED_ATTRIBUTES_SIGNING_KEY_ID_PARAM: !Sub "/${Environment}/ipv/core/sharedAttributes/signingKeyId"
+          SHARED_ATTRIBUTES_SIGNING_KEY_ID_PARAM: !Sub "/${Environment}/core/self/sharedAttributesJwtSigningKeyId"
       Policies:
         - DynamoDBReadPolicy:
             TableName: !Ref UserIssuedCredentialsTable
         - SSMParameterReadPolicy:
-            ParameterName: !Sub ${Environment}/ipv/core/sharedAttributes/*
+            ParameterName: !Sub ${Environment}/core/self/*
```

Comment:
In the spirit of having small lambdas each with limited scope of what it can access etc, then strictly speaking this lambda only needs to access `/${Environment}/core/self/sharedAttributesJwtSigningKeyId`, at present I don't think there's much else under that path but in future secure strings could be added for other purposes and perhaps worth considering tightening the scope?

Reply:
Yep, that makes sense to me! Will fix it up